### PR TITLE
Create Image.py file with image helper functions

### PR DESCRIPTION
**Pull request by courtneyambrozic**
In this pull request, the file structure is updated by removing the image.py file and creating a new Image class with the same functions.

### CODE REVIEW

**Comment by courtneyambrozic** [comment]
Hi @ptelang, sorry for the weird commit history here, but I updated the image functions in the format that we discussed. Let me know if you have any suggestions. Thanks!

File: cvpy/image/Image.py [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+''' Image analysis tools '''
+
+import sys
+import struct
+import numpy as np
+from typing import List
+from warnings import warn
+from swat import CAS, CASTable
+
+class Image(object):
```
Comment (ptelang) [inline cvpy/image/Image.py:28]: All of the methods in this class are static. So similar to __reverse, the remaining methods can also be annotated as staticmethod.

**Review by ptelang — COMMENTED** [review]
Hi Courtney, Thanks for refactoring the existing functions in image.py. I have one comment. Other than that, all looks good.

File: cvpy/image/Image.py (hunk 1 above)
Comment (courtneyambrozic) [reply cvpy/image/Image.py:28]: Thanks, Pankaj! I made this change for each of the functions. Let me know if you have any other comments.

**Pull request merged**